### PR TITLE
Docs: add basic info about 'LocalQueryRenderer' component

### PR DESCRIPTION
**Pull request by mrtnzlml**

### CODE REVIEW

**Comment by mrtnzlml** [comment]
Rebased. 🎸

File: docs/Modern-LocalStateManagement.md [hunk 1]
```diff
@@ -94,6 +94,8 @@ const Example = (props) => {
 }
 ```
 
+It is also possible to use `LocalQueryRenderer` component which works the same way like standard Query Renderer component except it doesn't send the request to the server. It's useful when you need to access _only_ local fields. Please note: Local Query Renderer returns only `props` in the `render` method - no errors (since there is no network request going on).
```
Comment (josephsavona) [inline docs/Modern-LocalStateManagement.md:97]: small grammar nit but otherwise looks great: "which works the same way as standard" (s/like/as)

Comment (mrtnzlml) [reply docs/Modern-LocalStateManagement.md]: @josephsavona Thanks! I need to work on my English. 😊

**Comment by mrtnzlml** [comment]
cc @kassens :)

**Pull request closed without merging**